revoke/revoke.go

```diff
@@ -22,14 +24,139 @@ import (
 	"github.com/cloudflare/cfssl/log"
 )
 
-// HardFail determines whether the failure to check the revocation
-// status of a certificate (i.e. due to network failure) causes
-// verification to fail (a hard failure).
-var HardFail = false
+type localCRL struct {
+	// embedded Mutex locker for the localCRL
+	sync.RWMutex
```

Comment:
Do you mean locking the whole structure just to read the `hardFail` value is ok? If yes, could you provide a proper example?